packages/react-native-reanimated/Common/cpp/reanimated/NativeAnimations/NativeAnimationValidation.cpp

```
@@ -0,0 +1,79 @@
+        !valueMatchesTarget(keyframe.value, track.target) || !isFinite(keyframe.timingToNext) ||
+        !hasOrderedTimingPoints(keyframe.timingToNext)) {
+      return TrackBuildFailure{TrackBuildFailureReason::InvalidValue};
+    }
+    previousOffset = keyframe.offset;
+  }
+  return std::nullopt;
+}
+
+} // namespace
+
+TrackBuildResult buildAnimationTrack(AnimationTrack track) {
```

Cppcheck (2.21.0)

[style] The function 'buildAnimationTrack' is never used.

(unusedFunction)